libethereum/Executive.cpp

```diff
@@ -418,6 +439,14 @@ bool Executive::go(OnOpFunc const& _onOp)
 			m_excepted = toTransactionException(_e);
 			revert();
 		}
+		catch (AddressAlreadyUsed const _e)
```

Comment:
Catch by const reference, then exception won't be copied here
`catch (AddressAlreadyUsed const& _e)`